mm/2s2h/DeveloperTools/SaveEditor.cpp

```diff
@@ -1390,6 +1413,49 @@ void GetPlayerForm(uint32_t form) {
     }
 }
 
+void ClearAllEquippedItems() {
+    auto& buttonItems = gSaveContext.save.saveInfo.equips.buttonItems;
+    auto& cButtonSlots = gSaveContext.save.saveInfo.equips.cButtonSlots;
+
+    for (size_t form = 0; form < ARRAY_COUNT(buttonItems); form++) {
+        for (size_t slot = 0; slot < ARRAY_COUNT(buttonItems[form]); slot++) {
+            if (slot == EQUIP_SLOT_B) {
+                // Form 0 is shared by FD and Human via CUR_FORM
+                // FD uses Deity Sword (not in B slot), Human uses regular swords
+                if (form == 0 && GET_PLAYER_FORM == PLAYER_FORM_FIERCE_DEITY) {
+                    // FD expects innate B action; leave untouched
+                    continue;
+                }
+                if (form != 0) {
+                    // Goron/Deku/Zora expect their innate B actions; leave untouched.
```

Comment:
Although the conditions are different, the reasoning and goal are the same, so let's condense this a little bit:
```suggestion
                // Goron/Deku/Zora/FD expect their innate B actions; leave untouched.
                // Form 0 is shared by FD and Human via CUR_FORM
                if (form != 0 || GET_PLAYER_FORM == PLAYER_FORM_FIERCE_DEITY) {
```